docs/src/development/release.mdx

```diff
@@ -17,7 +17,15 @@ If you are a developer on the MobX project, here are few guidelines that will he
 version: 0.2.0
 ```
 
-### 2. Changelog
+### 2. Expose version
+
+- Expose the updated version of `pubspec.yaml` to the library `.dart` file by running the appropriate command:
+
+```bash
+./tool/expose_version.dart <package_name>
```

Comment:
Since we only have 3 packages (as of now), we could just make this command as 

```sh
./tool/expose_version.dart mobx flutter_mobx mobx_codegen
```

Reply:
This part of the instructions is under the context of a single package, that's why I mentioned it like that. Otherwise, one should complete the first step of all packages and then run the multi-command.

The idea is that the changes of `expose_version.dart` should be included in the commit of the version bump and must run after the version update in `pubspec.yaml`.

Do I miss something?

Reply:
Yup sounds good!